features/server/test/capabilities.d/capabilities.list

```diff
@@ -1 +1,2 @@
-/usr/bin/ping cap_net_raw=ep
\ No newline at end of file
+# Issue 2530
+#/usr/bin/ping cap_net_raw=ep
```

Comment:
any specific reason to not delete the file instead? does it add value to keep this around in this form?